Example/TABTestKit.xcodeproj/project.pbxproj

```diff
@@ -307,7 +307,6 @@
 					};
 					70FEA77B1F6AC29400633C5C = {
 						CreatedOnToolsVersion = 8.3.3;
-						DevelopmentTeam = PR289YX2Y5;
```

Comment:
@annapiktas Apologies for the mess around but do you mind if you revert this change and the two below, please? As this wasn't the culprit for the build issue, I'm not sure what taking them away will do in the long run

Reply:
Np, thanks for catching this one.